Gemfile

```diff
@@ -18,7 +18,11 @@ gem 'pg'
 # OFN-maintained and patched version of Spree v2.0.4. See
 # https://github.com/openfoodfoundation/openfoodnetwork/wiki/Spree-2.0-upgrade
 # for details.
-gem 'spree', github: 'openfoodfoundation/spree', branch: '2-0-4-stable'
+gem 'spree_core', github: 'openfoodfoundation/spree', branch: '2-0-4-stable'
+gem 'spree_api', github: 'openfoodfoundation/spree', branch: '2-0-4-stable'
+gem 'spree_backend', github: 'openfoodfoundation/spree', branch: '2-0-4-stable'
+gem 'spree_frontend', github: 'openfoodfoundation/spree', branch: '2-0-4-stable'
+gem 'spree_sample', github: 'openfoodfoundation/spree', branch: '2-0-4-stable'
```

Comment:
I wonder what this is. Smells like something we can skip?

Reply:
yeah, I remove it in the final PR #3724